pkg/utils/k8utils.go

```diff
@@ -313,6 +313,12 @@ func (a *Apply) Apply(data []byte) error {
 	ioStreams := genericclioptions.IOStreams{In: os.Stdin, Out: os.Stdout, ErrOut: os.Stderr}
 	a.options = kubectlapply.NewApplyOptions(ioStreams)
 	a.options.DeleteFlags = a.deleteFlags("that contains the configuration to apply")
+	// This is a required field for server-side apply
+	a.options.FieldManager =  "application/apply-patch"
```

Comment:
Does this need to be `application/apply-patch+yaml`

Reply:
Its the expected content type for Yaml/JSON apply https://kubernetes.io/docs/reference/using-api/server-side-apply/#apply-and-update

Reply:
@LaVLaS Fixed this. Also updated the test image with this change